cluster_assign_roles.rb

```diff
@@ -221,6 +222,8 @@ def install_bootstrap(target_nodes)
                            search: 'role:BCPC-Hadoop-Head')
 
     target_head_nodes.each do |node|
+      puts "#{node[:fqdn]}: Permissioning head node"
```

Comment:
this should no longer be needed when we setup the permissions to look like the old chef-server right?

Plus this is only needed for VM clusters/ new clusters.

Reply:
For the current process, if you are creating new databag entries this is needed.